### PR TITLE
Merging with @quobit awesome-openbsd repo

### LINKED ISSUE #2: Offer to merge repos
Hi @ligurio !

I am the owner of [awesome-openbsd](https://github.com/quobit/awesome-openbsd) (CC0 licensed).

Some weeks ago I found your repo (as I noted on my own [issue](https://github.com/quobit/awesome-openbsd/issues/1)). I wonder if you would be interested in our repos being merged.

Regards.

### PR DESCRIPTION
**Pull request by quobit**
Hi @ligurio !

There's some more work to do (cleaning and reordering). But I think it is better to start doing this merge that fixes #2 and more PRs after that.

What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,162 +1,174 @@
...
-* [Vacancies](#vacancies)
-* [Hosting](#hosting)
-* [OpenBSD Free shells](#openbsd-free-shells)
-
-
-## OpenBSD-based products
-
-There are many products based on OpenBSD. Information about these products and the version of OpenBSD they are based on is often difficult to come by, since this fact is not widely publicised.
-
-##### OpenBSD-based commercial products
```
Comment (ligurio) [inline README.md:18]: Why did you remove this chapter?

File: README.md [hunk 2]
```diff
@@ -1,162 +1,174 @@
...
-
-- [Devio.us](http://devio.us/)
-- [Polarhome](http://www.polarhome.com/)
-- [Grex](http://grex.org/)
-- [Silenceisdefeat](http://silenceisdefeat.com/)
-- [Free Shell Accounts](http://shells.red-pill.eu/)
-- [Underlegend Networks](http://yenn.ulegend.net/)
-- [obsd.pl](http://obsd.pl/)
-
-### Interviews
```
Comment (ligurio) [inline README.md]: This section removed too. Why?

File: README.md [hunk 3]
```diff
@@ -1,162 +1,174 @@
...
-* Henning Brauer [IOException.de](http://tmp.marmaro.de/www.ioexception.de/2013/10/16/interview-henning-brauer/index.html)
-* Mike Larkin [bsdtalk](http://bsdtalk.blogspot.ru/2010/08/bsdtalk195-mike-larkin.html)
-* Anil Madhavapeddy [FOSDEM](https://archive.fosdem.org/2012/interview/anil-madhavapeddy.html)
-* Pierre-Yves Ritschard [bsdtalk](http://bsdtalk.blogspot.ru/2007/02/bsdtalk097-openbsd-developer-pierre.html)
-* Claudio Jeker [bsdtalk](http://bsdtalk.blogspot.ru/2007/01/bsdtalk095-openbsd-developer-claudio.html)
-* Jason Wright [bsdtalk](http://bsdtalk.blogspot.ru/2006/11/bsdtalk082-openbsd-developer-jason.html)
-* Marc Balmer [bsdtalk](http://bsdtalk.blogspot.ru/2006/10/bsdtalk076-openbsd-developer-marc.html)
-* Matthieu Herrb [bsdtalk](http://bsdtalk.blogspot.ru/2007/04/bsdtalk106-interview-with-matthieu.html), [bronevichok.ru](https://bronevichok.ru/blog/2014/08/06/testing-of-xorg.html)
-
-### Desktop
```
Comment (ligurio) [inline README.md:147]: Why have you removed it?

File: README.md [hunk 4]
```diff
@@ -1,162 +1,174 @@
...
+* [Getting OpenBSD running on Raspberry Pi 3](http://undeadly.org/cgi?action=article&sid=20170409123528)
+* [A simple first server](http://blog.hermes-technology.de/openbsd/server/2017/06/06/a-first-server.html) - A series of posts about OpenBSD server configuration for learning purposes
+* [OpenBSD manpages reading list](https://gist.github.com/QWxleA/0a3e28f4a3387e5087e8f3608c32fd03)
+* [OpenBSD porting workshop, August 11, 2018](https://www.twitch.tv/videos/296003844) - twitch.tv video by bcallah@
+* [Debian on OpenBSD vmd (without qemu or another debian system)](http://www.netzbasis.de/openbsd/vmd-debian/)
+* [Support of OpenBSD pledge(2) in programming languages](https://gist.github.com/ligurio/f6114bd1df371047dd80ea9b8a55c104)
+* [Setup environment in OpenBSD using Ansible playbook](https://github.com/ligurio/openbsd-cookbooks)
+
+ 
+## Blogs
```
Comment (ligurio) [inline README.md]: There is an Planet OpenBSD for blogs related to OpenBSD - https://github.com/ligurio/openbsdnow.org/ ([web version](https://openbsdnow.org/))
Most of the blogs from this section already there.

File: README.md [hunk 5]
```diff
@@ -1,162 +1,174 @@
...
+
+## Twitter
+
+* [@OpenBSD](https://twitter.com/OpenBSD)
+* [@OpenBSD_CVS](https://twitter.com/OpenBSD_CVS) - OpenBSD Commit messages in 140 characters or less. For individual modules, see: [@OpenBSD_src](https://twitter.com/OpenBSD_src), [@OpenBSD_ports](https://twitter.com/OpenBSD_ports), [@OpenBSD_www](https://twitter.com/OpenBSD_www), [@OpenBSD_xenocar](https://twitter.com/OpenBSD_xenocar), [@OpenBSD_stable](https://twitter.com/OpenBSD_stable), [@OpenBSD_sets](https://twitter.com/OpenBSD_sets) (all built by [Andrew Fresh](https://twitter.com/afresh1))
+* [@OpenBSDJournal](https://twitter.com/openbsdjournal) - tweets to Undeadly.org stories
+* [@OpenSMTPD](https://twitter.com/opensmtpd)
+* [@OpenBSDNow](https://twitter.com/openbsdnow) - OpenBSD News & Updates
+* [@mpotd_openbsd](https://twitter.com/mpotd_openbsd) - Man Page of The Day
+* [@romanzolotarev](https://twitter.com/romanzolotarev) - lots of good advice
```
Comment (ligurio) [inline README.md]: As far as I understand this section contains openbsd-related twitter accounts. Last one is a personal twitter account of Roman Zolotarev. He is not an OpenBSD developer, not a person affiliated with OpenBSD project. Why we should add him to a list?

File: README.md [hunk 6]
```diff
@@ -1,162 +1,174 @@
...
+
+## Mastodon ([bsd.network](https://bsd.network/) instance)
+
+* [@phessler](https://bsd.network/@phessler) - bsd.network's admin
+* [@AFresh1](https://bsd.network/@AFresh1)
+* [@bcallah](https://bsd.network/@bcallah)
+* [@romanzolotarev](https://bsd.network/@romanzolotarev)
```
Comment (ligurio) [inline README.md:221]: The  same, see previous comment.

File: README.md [hunk 7]
```diff
@@ -1,162 +1,174 @@
...
+## Telegram Groups
+
+* [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8Mcj89LYcDGKiQ) from the [OpenBSD Jumpstart](http://www.openbsdjumpstart.org/) author
+* [OpenBSD Brazil](https://t.me/OpenBSDbr)
+* [OpenBSD Spain](https://t.me/OpenBSD_es)
+
+## Third Party repositories
+
+* [Scripts to run an OpenBSD mirror](https://github.com/bluhm/mirror-openbsd)
+* [OpenBSD work in progress ports](https://github.com/jasperla/openbsd-wip)
```
Comment (ligurio) [inline README.md]: There are many openbsd-wip repositories and japserla/openbsd-wip is only one of them. I suggest to add a link to a list of repositories with topic "openbsd-wip" - https://github.com/topics/openbsd-wip Repository owned by jasper is already there.

**Review by ligurio — COMMENTED** [review]
Hi, Jose! Looks good at first glance. Thanks! 
Take a look at my comments inside commits.

File: README.md (hunk 1 above)
Comment (quobit) [reply README.md:18]: It hasn't been removed but relocated.

File: README.md (hunk 2 above)
Comment (quobit) [reply README.md]: the same as above

File: README.md (hunk 3 above)
Comment (quobit) [reply README.md:147]: idem

File: README.md (hunk 4 above)
Comment (quobit) [reply README.md]: Yes, there's still duplicates to remove in future PRs.

File: README.md (hunk 5 above)
Comment (quobit) [reply README.md]: You're right. I'll remove it

File: README.md (hunk 6 above)
Comment (quobit) [reply README.md:221]: I think Roman is doing a great job helping and spreading OpenBSD with his blog and social network accounts. IMHO, he's clearly part of the community.

File: README.md (hunk 7 above)
Comment (quobit) [reply README.md]: Great! Good idea! Done!

**Comment by ligurio** [comment]
Well, now it looks a bit better. I'll merge it, but could you combine all these commits into a single one? (```git rebase -i``` with squash can help) and make commit message more clear?

**Comment by quobit** [comment]
@ligurio done!

**Comment by ligurio** [comment]
Cool! Thank you, Jose!
